src/AddTodoForm.jsx

```diff
@@ -1,8 +1,16 @@
+/* eslint-disable no-undef */
 // eslint-disable-next-line no-unused-vars
 import React from 'react';
 
+
+
+
 // eslint-disable-next-line react/prop-types
 const AddTodoForm = ({ onAddTodo }) => {//onAddTodo prop, passed from App, to add a new todo
+  
+  const [todoTitle, setTodoTitle] = useState(''); 
+
+  
   const handleAddTodo = (event) => {
     event.preventDefault();
     const todoTitle = event.target.elements.title.value.trim();
```

Comment:
I think you may have overlooked this in the homework as well:

![image](https://github.com/user-attachments/assets/127e16c7-7732-42d8-83a8-7c6b24cac91c)


please refactor 👍